manifests/dispatch_router.pp

```diff
@@ -7,6 +7,17 @@
 
   class { 'qpid::router': }
 
+  if $monitor_qpid_router {
```

Comment:
I assume this should be in $capsule::monitor_qpid_router? Ditto for $monitor_interval